src/app/stacktrace_win_dlg.h

```diff
@@ -60,7 +60,11 @@ class StraceDlg : public QDialog, private Ui::errorDialog
             "</p></font>"
             "<br/><hr><br/>"
             "<p align=center><font size=4>"
-            "qBittorrent version: " VERSION "<br/>"
+            "qBittorrent version: " VERSION
+#ifdef Q_OS_WIN64
+                              " (64 bit)"
```

Comment:
I believe `" (64-bit)"` should be used here as well.  See my other comment for details.